src/Explorer/Controls/NotebookGallery/Cards/GalleryCardComponent.tsx

```diff
@@ -138,6 +157,7 @@ export class GalleryCardComponent extends React.Component<GalleryCardComponentPr
   private generateIconButtonWithTooltip = (
     iconName: string,
     title: string,
+    rightAlign: boolean,
```

Comment:
super-nit: (I'm totally splitting hair here, so feel free to ignore :)): you could replace with `horizontalAlign: "left"|"right`. Technically, `rightAlign` being `false` does not necessarily mean left-aligned.